ios/Plugin/Plugin.swift

```diff
@@ -386,3 +396,32 @@ public class NativeBiometric: CAPPlugin {
         guard status == errSecSuccess || status == errSecItemNotFound else { throw KeychainError.unhandledError(status: status) }
     }
 }
+
+extension LAContext {
+    static var savedBiometricsPolicyState: Data? {
+        get {
+            UserDefaults.standard.data(forKey: "BiometricsPolicyState")
+        }
+        set {
+            UserDefaults.standard.set(newValue, forKey: "BiometricsPolicyState")
+        }
+    }
+    
+    static func biometricsChanged() -> Bool {
+        let context = LAContext()
+        var error: NSError?
+        context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: &error)
+        
+        // If there is no saved policy state yet, save it
+        if error == nil && LAContext.savedBiometricsPolicyState == nil {
+            LAContext.savedBiometricsPolicyState = context.evaluatedPolicyDomainState
+            return false
```

Comment:
About this, so this will be called when we're trying to save a new biometrics from no biometrics right?

How about making this return true? Since we're changing from no biometrics to have biometrics.
But this is optional, it's okay if you think it's better to make it still returning false;

Reply:
I think it's only happening when user enrolls the biometry for the first time, so it's safe to return false. What do you think, mas?

Reply:
Okay, that's also logical since it's not a "change" but "addition". I see, thank you